<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorScopeConfigWidget</class>
 <widget class="QWidget" name="MotorScopeConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="dataChannelLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Data Channel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="dataChannelSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="triggerChannelLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Trigger Channel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="triggerChannelSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="verticalRangeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Vertical Range</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="verticalRangeDoubleSpinBox">
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="suffix">
      <string> V</string>
     </property>
     <property name="maximum">
      <double>20.000000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="recordLengthLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Record Length</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="recordLengthSpinBox">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> Sa</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="value">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="sampleIntervalLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sample Interval</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="sampleIntervalDoubleSpinBox">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string> ns/Sa</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>999999999999.000000000000000</double>
     </property>
     <property name="value">
      <double>500.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="triggerDirectionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Trigger Direction</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="triggerDirectionComboBox">
     <item>
      <property name="text">
       <string>Rising</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Falling</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="sampleRateDoubleSpinBox">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> Sa/sec</string>
     </property>
     <property name="maximum">
      <double>999999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="recordTimeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Record Time</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="recordTimeDoubleSpinBox">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="sampleRateLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sample Rate</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
